### DISCUSSION TITLE
For some reason code passes cloudflare on nowsecure.nl and doesn't pass on some other websites.

### DISCUSSION THREAD

**Discussion by MalvinJohnas** [seleniumbase/SeleniumBase#2167] [Q&A] [answered]
driver = Driver(undetectable=True, headless=False, proxy=proxy)
    print(proxy)
    
    #driver.execute_script("window.open('http://www.google.com/', '_blank');")
    sleep(random.uniform(1.5, 2))
    #driver.switch_to.window(driver.window_handles[-1])

    driver.get(url)
    sleep(random.uniform(1, 2))

    WebDriverWait(driver, 20).until(EC.frame_to_be_available_and_switch_to_it((By.CSS_SELECTOR,"iframe[title='Widget containing a Cloudflare security challenge']")))
    sleep(random.uniform(1.5, 2))
    WebDriverWait(driver, 20).until(EC.element_to_be_clickable((By.CSS_SELECTOR, "label.ctp-checkbox-label"))).click()  

    page_source = driver.page_source

    sleep(30)
    driver.quit()

    
    
For some reason, this code passes the check on NowSecure but doesn't on other websites. However, if I open a new tab using the mouse and paste the link to the website, I pass the check. If I click the check by hand, opened by the script, it doesn't pass.

"you are using an unsupported command-line flag ignore-certificate-errors. Stability and security will suffer"
Additionally, I noticed that when the NowSecure website opens, the warning on the top of the screen disappears, and if I go to another website, it stays there.

Based on these tests, I don't think the problem is with the proxy, as I am able to pass it by hand. Probably, the mistake is somewhere in driver.get(url).

**Comment by mdmintz** [comment, marked as answer]
It looks like you're doing some odd things in your script. Maybe you should check out the examples:

### Here are some existing examples that use ``--uc`` mode:
* https://github.com/seleniumbase/SeleniumBase/blob/master/examples/verify_undetected.py
* https://github.com/seleniumbase/SeleniumBase/blob/master/examples/uc_cdp_events.py
* https://github.com/seleniumbase/SeleniumBase/blob/master/examples/raw_uc_mode.py

### Here's a raw driver example:

```python
from seleniumbase import Driver
import time

driver = Driver(uc=True)
driver.get("https://nowsecure.nl/#relax")
time.sleep(6)
driver.quit()
```

--------

The `"you are using an unsupported command-line flag ignore-certificate-errors. Stability and security will suffer"` message is harmless. The option to hide it would make your browser detected.

There's `driver.uc_click(selector)` if you need to click on something. The point of UC Mode is to prevent your browser from having to go through that checkpoint in the first place.

With SeleniumBase, you shouldn't need to use `WebDriverWait` at all... automatic waiting is already included with methods, such as `driver.click(selector)`, etc. Stick with the examples for understanding how to create scripts.
